internal/pb/piece.pb.go

```diff
@@ -0,0 +1,164 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.27.1
```

Comment:
i thought that this module (proto) can be used by storage node and gateway node. Shouldn't it be in 'pkg' instead of 'internal'?

Reply:
Internal can be used from any our module/submodule, just for public usage will not be able

Reply:
but for storage and gateway node it's a public usage, no?

Reply:
Now I see issue... moved to assets/pb package for using in gateway and storage nodes.

Reply:
Now I see issue... moved to /pb package